### PR TITLE
#1029 structure shouldnt flip if part selected

### PR DESCRIPTION
**Pull request by Alex-Work-Account**

### CODE REVIEW

File: packages/ketcher-core/src/application/editor/actions/rotate.ts [hunk 1]
```diff
@@ -34,26 +34,37 @@ export function fromFlip(restruct, selection, dir, center) {
 
   const action = new Action()
 
-  if (!selection) selection = structSelection(struct)
+  if (!selection) {
+    selection = structSelection(struct)
+  }
```
Comment (r1z3rISGOD) [inline packages/ketcher-core/src/application/editor/actions/rotate.ts:39]: Why is this needed?

Comment (Alex-Work-Account) [reply packages/ketcher-core/src/application/editor/actions/rotate.ts:39]: I think it's much more readable, can't miss the brackets :)

**Review by AndreiMazol — APPROVED** [review]
